docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -143,6 +143,16 @@ BigCommerce sends requests for tax estimates on product item, shipping, and hand
 
 [View the Estimate Taxes API reference](/docs/rest-contracts/tax#estimate-taxes).
 
+### Performance
+
+Merchants expect fast tax estimate responses to facilitate a streamlined checkout experience for their shoppers, hence performance is an essential consideration for a tax provider.
```

Comment:
One common issue we've explored with slow tax providers is whether they are conducting additional lookup operations _during a tax calculation_.

For example, actually calling out to BC for more information.

The most essential consideration there would be that _all the data needed to conduct a tax calculation should be in the tax request itself_. If a tax provider is calling out to BigCommerce for more data this is a signal they need to raise a feature request to include additional information in the request OR they need to source the data by other means (e.g. tax properties).

Obviously we can only guide so far and in so much detail - hence we are highlighting the core performance goal here only.

Reply:
🤔 Perhaps a refined stance above is:

_all the data needed to conduct a tax calculation should be in the tax request itself_

...becomes...

_all the BC data needed to conduct a tax calculation should be in the tax request itself_